Library/Homebrew/cli/named_args.rb

```diff
@@ -38,6 +38,14 @@ def to_formulae_and_casks(only: nil, method: nil)
         end
       end
 
+      def to_formulae_casks_unknowns(method: nil)
```

Comment:
Does anyone have suggestions for a method name? I'm not overly satisfied with the current one; `to_formulae_and_casks_and_unknowns` is what the naming scheme would suggestion, but it's a bit long, but maybe worth the consistency

Reply:
`unavailable_formula_and_casks`? Is it returning an array of the failure exceptions?

Reply:
It is: we need the failures for when a user enters a recently deleted formula

Reply:
`to_formulae_and_casks_and_unavailable`, `to_formulae_and_casks_failures`, `to_formulae_and_casks_exceptions`?

Reply:
`to_formulae_and_casks_and_unavailable` I think is the best one since it's the most consistent with the other methods.

I'm thinking it might be worthwhile to refactor `named_args`... been thinking about possible designs but this is not a great place to discuss it I suppose

Reply:
> `to_formulae_and_casks_and_unavailable` I think is the best one since it's the most consistent with the other methods.

👍🏻 


> I'm thinking it might be worthwhile to refactor `named_args`... been thinking about possible designs but this is not a great place to discuss it I suppose

Go for it! Feel free to open a semi-broken draft PR to discuss 👍🏻